lib/ClangImporter/ClangImporter.cpp

```diff
@@ -984,6 +984,21 @@ ClangImporter::getClangArguments(ASTContext &ctx) {
   return invocationArgStrs;
 }
 
+/// Gather all -Xclang flags from a Clang driver command-line
+static void extractClangFrontendInvocationArgs(const ArrayRef<std::string> invocationArgStrs,
+                                               std::vector<const char *> &clangFrontendArgs) {
+  clangFrontendArgs.push_back(invocationArgStrs.front().c_str());
```

Comment:
Can you point out where this is being set to something looking like a compiler executable? Asking because I don't see that being specified in the test invocation and curious whether we should check it at least _looks_ like one.

Reply:
`ClangImporter::getClangArguments` is the source of `invocationArgStrs` and it makes sure to begin with an executable path; and, the original downstream code also made an assumption that `[0]` is an executable path, in `clang::createInvocationFromCommandLine`.

But it won't hurt to add an assert here, at least, to verify that this looks like a `clang` executable. 👍🏼

Reply:
The above simplification made this code redundant so I removed it.